actions/configure-runner/README.md

```diff
@@ -14,7 +14,7 @@ Earthly to consume them correctly.
 
 ```yaml
 - name: Setup Remote Runner
-  uses: input-output-hk/catalyst-ci/actions/configure-runner@master
+  uses: input-output-hk/catalyst-ci/actions/configure-runner@fdcDeprecation
```

Comment:
There is no need to change this since it is a martdown file, but it is okay, just remember to change it back to